src/mlpack/core/util/cli.hpp

```diff
@@ -2,7 +2,7 @@
  * @file cli.hpp
```

Comment:
Once we change it to its final name, we should actually rename the file with git too. :)

Reply:
I will rename everything back to `IO` even file name.